docs/installation/installation-troubleshooting.md

```diff
@@ -0,0 +1,94 @@
+Shopsys Framework normalizes the names of locales present in different systems by creating new collations in the database.
+
+However, if your operating system does not provide the required locales you can try:
+* On Linux: Install additional locales to your system (eg. on Debian Linux this can be done by installing [locales-all](https://packages.debian.org/cs/stable/locales-all) package) and restart the database server.
+* On Windows: Make sure you use PostgreSQL distribution that supports multiple locales. We recommend to use [EnterpriseDB PostgreSQL distribution](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads#windows).
+* Otherwise: The only other option is to create the database collation mentioned in the exception manually using a locale that your OS supports.
+(Note: every OS should support special locale `"C"` or `"POSIX"`.)
+
+## Phing target timezones-check fails
+
+Error message:
+```
+[Shopsys\FrameworkBundle\Command\Exception\DifferentTimezonesException]
+  Timezones in PHP and database configuration must be identical. Current settings - PHP:UTC, PostgreSQL:Europe/Prague
+```
+
+The problem is that your `timezone` setting in PostgreSQL and `date.timezone` in `php.ini` are different.
+
+Currently, some features are dependent on the fact that time zones in database and PHP are the same. Please set them both to the same timezone.
+
+Timezone used in PostgreSQL can be determined by running the following command:
+```
+psql --username postgres --dbname <database_name> --command "SHOW timezone"
+```
+
+Timezone for PHP can be set in your `php.ini` (usually located in `/etc/php.ini`) by configuration like:
+```
+date.timezone = "UTC"
+```
+
+## Phing target npm fails
+
+In some cases we encountered that node packages failed to install locally.
+Nevertheless installation was able to process via global parameter.
+```
+npm install --global <path-to-package.json>
+```
+
+## There are no logs during installation or use of application
+
+Monolog is configured to log into streams in [`app/config/packages/monolog.yml`](../../project-base/app/config/packages/monolog.yml).
+If you want to log into a file change the configuration of handlers like this:
+```yaml
+monolog:
+  # ...
+  handlers:
+    # ...
+    cron:
+      type: rotating_file
+      max_files: 7
+      path: "%kernel.logs_dir%/%kernel.environment%.cron.log"
+```
+
+## Composer dependencies installation fails on memory limit
+When `composer install` or `composer update` fails on an error with exceeding the allowed memory size, you can increase the memory limit by setting `COMPOSER_MEMORY_LIMIT`.
+
+## There is not possible to create, copy, move or remove files on local filesystem (Windows like filesystems)
+When there is not possible to do some operations with files withing labeled local filesystem (`C:`, `D:`, ...), the problem could be solved by removing label part from the path to the files.
```

Comment:
I think it would be better to add that you can use `TransformString::removeDriveLetterFromPath` to remove it